packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -224,6 +234,22 @@ class MessageList extends Widget<Properties> {
     }
   }
 
+  _isTimeoutExceeded(lastMessage: Message, newMessage: Message): boolean {
+    const lastMessageTimestamp = lastMessage?.timestamp;
+    const newMessageTimestamp = newMessage?.timestamp;
+
+    if (!lastMessageTimestamp || !newMessageTimestamp) {
+      return false;
+    }
+
+    const lastMessageTimestampInMs = new Date(lastMessageTimestamp).getTime();
```

Comment:
It’s safer to use the dateSerialization.deserializeDate(value) function to convert many possible formats of the incoming timestamp option value to a Date format

Reply:
Okay, I get it